templates/Categories/add.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @var \App\View\AppView $this
+ * @var \ItemManager\Model\Entity\Category $category
+ */
+?>
+<div class="row">
+    <aside class="column">
+        <div class="side-nav">
+            <h4 class="heading"><?= __('Actions') ?></h4>
+            <?= $this->Html->link(__('List Categories'), ['action' => 'index'], ['class' => 'side-nav-item']) ?>
+        </div>
+    </aside>
+    <div class="column-responsive column-80">
+        <div class="categories form content">
+            <?= $this->Form->create($category) ?>
+            <fieldset>
+                <legend><?= __('Add Category') ?></legend>
+                    <?= $this->Form->control('name'); ?>
```

Comment:
Related to https://github.com/MattTyrell/tutorial-plugin/pull/1#discussion_r1914527398.
```suggestion
                    <?= $this->Form->control('name') ?>
```
Please fix for other places.

Reply:
Done and fixed as per bfc8aca736fcd9e7b73adbe8191c62dc0733c53a.